pandas/io/json/normalize.py

```diff
@@ -149,30 +169,40 @@ def json_normalize(data, record_path=None, meta=None,
     1  NaN         NaN      Regner        NaN       Mose       NaN
     2  2.0  Faye Raker         NaN        NaN        NaN       NaN
 
+    >>> from pandas.io.json import json_normalize
```

Comment:
can you give the same example w/o max_level as well

Reply:
It will be the same w/o max_level, whereas max_level=0 will have a different effect. Will that work?

Reply:
sure